FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -101,26 +108,54 @@ - (instancetype)initWithDatabaseManager:(RCNConfigDBManager *)manager
                                                                   bundleID:_bundleIdentifier
                                                                  namespace:_FIRNamespace];
     _isFetchInProgress = NO;
+
+    // Check if app group is set. If so, use it to communicate the latest fetch time.
+    if (options.appGroupID) {
+      _sharedUserDefaultsSuiteName = options.appGroupID;
+      /// Instantiate the shared instance.
+      _sharedUserDefaults = [self sharedUserDefaults];
+    }
   }
   return self;
 }
 
+- (NSUserDefaults *)sharedUserDefaults {
+  static dispatch_once_t onceToken;
+  static NSUserDefaults *sharedInstance;
+  dispatch_once(&onceToken, ^{
+    sharedInstance = [[NSUserDefaults alloc] initWithSuiteName:self->_sharedUserDefaultsSuiteName];
+  });
+  return sharedInstance;
+}
+
 #pragma mark - read from / update userDefaults
 - (NSString *)lastETag {
   return [_userDefaultsManager lastETag];
 }
 
 - (void)setLastETag:(NSString *)lastETag {
   [_userDefaultsManager setLastETag:lastETag];
```

Comment:
The intention here then is we write to both?

Reply:
that's right. The _userDefaultsManager writes to the existing suite (group.{bundleID}.firebase) and data there is not meant to be shared outside the process (app).